Helper/FieldValidationHelper.php

```diff
@@ -138,12 +139,12 @@ private function validateIntegrationRequiredFields(Form $fieldMappingsForm, arra
      *
      * @return array
      */
-    private function findMissingIntegrationRequiredFieldMappings(string $object, array $mappedFields)
+    private function findMissingIntegrationRequiredFieldMappings(string $object, array $mappedFields): array
     {
         $requiredFields = $this->integrationObject->getRequiredFieldsForMapping($object);
 
         $missingFields = [];
-        foreach ($requiredFields as $field => $label) {
+        foreach ($requiredFields as $field => $fieldObjest) {
```

Comment:
Not so important because it's not used, unless you meant `jest` as in satire :)

```suggestion
        foreach ($requiredFields as $field => $fieldObject) {
```

Reply:
Thanks, I did not notice :) Updated.
BTW nice thing that "Suggested change".